ex4/tema4.js

```diff
@@ -0,0 +1,14 @@
+
+
+
+  function addNumber(...theArgs)
+         {
+            let sum = 0;
+            for (const arg of theArgs)
+            {
+               sum += arg;
+            }
+            return sum;
+         }
```

Comment:
se poate de lucrat la identare?

paranteza `}` sa fie in prima linie. Si fiecare tab nou sa aiba 4 caractere